sqlite_utils/db.py

```diff
@@ -930,6 +930,43 @@ def analyze(self, name=None):
             sql += " [{}]".format(name)
         self.execute(sql)
 
+    def init_spatialite(self, path: str) -> bool:
```

Comment:
I still like the idea of making `path` here optional - if you don't provide it (or set `path=None`) then the function can call `find_spatialite()` for you.

Reply:
Done.